test/EnsembleKalmanProcess/runtests.jl

```diff
@@ -158,7 +165,7 @@ const EKP = EnsembleKalmanProcesses
         eki_final_result = vec(mean(get_u_final(ekiobj), dims = 2))
 
         # kind of arbitrary tolerance here,
-        @test norm(u_star - eki_final_result) < 0.1
+        @test norm(u_star - eki_final_result) < 0.15
```

Comment:
This test is the reason CI [failed on Ubuntu](https://github.com/CliMA/EnsembleKalmanProcesses.jl/runs/4909239465?check_suite_focus=true#step:6:172) in previous commit; missed this because I don't have an Ubuntu box to test on.

Based on the comment, I'm increasing the tolerance so that the test passes. For the record, in the earlier version of these tests (EnsembleKalmanProcessModule/runtests.jl) this test was commented out, and the tolerance was set to 0.5.

Reply:
Mmmmh this seems to still be a problem sometimes. Honestly, this was a bad test to begin with. Could you instead compare
`@test norm(u_star - eki_final_result) < norm(u_star - eki_init_result)`?

where
eki_init_result = vec(mean(get_u_prior(ekiobj), dims = 2))

Reply:
Thanks, that's a better idea than just bumping the threshold!

I'll continue squashing commits onto this branch and copying/cherry-picking them onto `tsj/add-explicit-rng` #81, to have a paper trail -- let me know if you want me to proceed differently. Apologies for making this messier than it needed to be: I understand that you want to have the PR reviewed and finalized *before* the squash... 🙁

Reply:
Done in b55846d